my-app/src/components/Test.vue

```diff
@@ -1,24 +1,46 @@
 <template>
-  <p>Users data</p>
+  <p>Users data: </p>
   <p>{{ data }}</p>
+  <div v-if="isError">
+    Something went wrong: {{error.message}}
+  </div>
 
-  <p>Create new user</p>
-  <form @submit.prevent="doSomething">
-    <div><input v-model="name" /></div>
-    <div><input v-model="email" /></div>
-
+  <br>
+  <p>Create new user: </p>
+  <form @submit.prevent="addUser">
+    <div><label>Name <input v-model="name" /></label></div>
+    <div><label>email <input v-model="email" /></label></div>
```

Comment:
Nitpick:
No need for these divs, right? 😄 
```suggestion
    <label>Name <input v-model="name" /></label>
    <label>email <input v-model="email" /></label>
```